src/pages/ni/auxiliaries/Dashboard.vue

```diff
@@ -343,6 +339,18 @@ export default {
     },
     async removeElementFromFilter (sector) {
       this.filteredSectors = this.filteredSectors.filter(sec => sec !== sector._id);
+      delete this.displayStats[sector._id];
+    },
+    setDisplayStats (data, fullRefresh) {
+      for (const sector of this.filteredSectors) {
+        if (!this.displayStats[sector] || this.displayStats[sector].loading ||
+          (fullRefresh && !this.displayStats[sector].loading) || Object.keys(data).includes('loadingDetails')) {
```

Comment:
tu n'as pas besoin de la condition `!this.displayStats[sector].loading` car tu as deja `this.displayStats[sector].loading` dans le bloc d'avant